scripts/deploy.sh

```
@@ -0,0 +1,35 @@
+
+echo "🛑 기존 프로세스 종료"
+if pgrep -f "$JAR_NAME" > /dev/null; then
+  pkill -f "$JAR_NAME"
+  echo "→ 프로세스 종료 시도 중..."
+  sleep 2  # 종료되기까지 잠깐 대기
+else
+  echo "→ 종료할 프로세스 없음"
+fi
+
+echo "🗂 JAR 복사"
+LATEST_JAR=$(ls -t $BUILD_DIR/*.jar | grep -v 'plain' | head -n 1)
```

Shellcheck (0.10.0)

[warning] Don't use ls | grep. Use a glob or a for loop with a condition to allow non-alphanumeric filenames.

(SC2010)